src/main/java/core/basesyntax/FruitDB.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class FruitDB {
+    private final Map<String, Integer> inventory = new HashMap<>();
+
+    public void add(String fruit, int quantity) {
+        inventory.put(fruit, inventory.getOrDefault(fruit, 0) + quantity);
+    }
+
+    public void subtract(String fruit, int quantity) {
+        inventory.put(fruit, inventory.getOrDefault(fruit, 0) - quantity);
```

Comment:
Consider adding a check to prevent the inventory from having negative quantities. You might want to throw an exception or handle the case where the subtraction would result in a negative quantity.